src/devices/owon.ts

```diff
@@ -126,7 +126,20 @@ const fzLocal = {
         },
     } satisfies Fz.Converter,
 };
+const tzLocal = {
+    PC321_clearMetering: {
+        key: ['clear_metering'],
+        convertSet: async (entity, key, value, meta) => {
+            const endpoint = meta.device.getEndpoint(1);
+            const group = 0xFFE0;
+            const command = 0x00;
+            const payload = {};
 
+            await endpoint.command(group, command, payload, {disableDefaultResponse: true});
```

Comment:
Could you check if publish to `zigbee2mqtt/DEVICE/set`, payload = `{"reset": ""}` also works?

Reply:
I mean publishing to MQTT topic: `zigbee2mqtt/DEVICE_NAME/set` payload `{"reset":""}` (without making any changes)

Reply:
For me, it doesn't work.

![image](https://github.com/user-attachments/assets/9f70b2ce-e3fa-4d5e-856c-3b1d70631d35)

Reply:
What do you see in the debug log when sending this?

Reply:
![image](https://github.com/user-attachments/assets/9921ed6c-dc14-40cf-ae3e-97e528059869)

Reply:
The debug log is only visible in the files, see [this](https://www.zigbee2mqtt.io/guide/configuration/logging.html) on how to enable debug logging.

Reply:
![image](https://github.com/user-attachments/assets/1ff1a56c-0433-4a48-8775-a63e74235f94)
![image](https://github.com/user-attachments/assets/ea5249c6-8a08-4ceb-a95c-cc0f66676224)
![image](https://github.com/user-attachments/assets/3c2abea4-2f01-4d33-a37f-b56409d08a8c)


`debug 2024-09-08 21:39:21z2m:mqtt: Received MQTT message on 'zigbee2mqtt/0x3c6a2cfffed3a374/set' with data '{"reset":""}'
debug 2024-09-08 21:39:21z2m: Publishing 'set' 'reset' to '0x3c6a2cfffed3a374'
debug 2024-09-08 21:39:21zh:controller:endpoint: ZCL command 0x3c6a2cfffed3a374/1 genBasic.resetFactDefault({}, {"timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"reservedBits":0,"writeUndiv":false})
debug 2024-09-08 21:39:21zh:ember: ~~~> [ZCL to=15414 apsFrame={"profileId":260,"clusterId":0,"sourceEndpoint":1,"destinationEndpoint":1,"options":4416,"groupId":0,"sequence":0} header={"frameControl":{"reservedBits":0,"frameType":1,"direction":0,"disableDefaultResponse":false,"manufacturerSpecific":false},"transactionSequenceNumber":43,"commandIdentifier":0}]
debug 2024-09-08 21:39:21zh:ember:ezsp: ===> [FRAME: ID=52:"SEND_UNICAST" Seq=80 Len=24]
debug 2024-09-08 21:39:21zh:ember:uart:ash: ---> [FRAME type=DATA frmTx=0 frmRx=7](ackRx=0)
debug 2024-09-08 21:39:21zh:ember:uart:ash: <--- [FRAME type=DATA]
debug 2024-09-08 21:39:21zh:ember:uart:ash: <--- [FRAME type=DATA ackNum=1](ackRx=0 frmTx=1)
debug 2024-09-08 21:39:21zh:ember:uart:ash: <--- [FRAME type=DATA ackNum=1 frmNum=7](frmRx=7) Added to rxQueue
debug 2024-09-08 21:39:21zh:ember:uart:ash: ---> [FRAME type=ACK frmRx=0](ackRx=1)
debug 2024-09-08 21:39:21zh:ember:ezsp: <=== [FRAME: ID=52:"SEND_UNICAST" Seq=80 Len=7]
debug 2024-09-08 21:39:21zh:ember:ezsp: ~~~> [SENT type=DIRECT apsSequence=166 messageTag=43 status=OK]
debug 2024-09-08 21:39:21zh:ember:uart:ash: <--- [FRAME type=DATA]
debug 2024-09-08 21:39:21zh:ember:uart:ash: <--- [FRAME type=DATA ackNum=1](ackRx=1 frmTx=1)
debug 2024-09-08 21:39:21zh:ember:uart:ash: <--- [FRAME type=DATA ackNum=1 frmNum=0](frmRx=0) Added to rxQueue
debug 2024-09-08 21:39:21zh:ember:uart:ash: <--- [FRAME type=DATA]
debug 2024-09-08 21:39:21zh:ember:uart:ash: <--- [FRAME type=DATA ackNum=1](ackRx=1 frmTx=1)
debug 2024-09-08 21:39:21zh:ember:uart:ash: <--- [FRAME type=DATA ackNum=1 frmNum=1](frmRx=1) Added to rxQueue
debug 2024-09-08 21:39:21zh:ember:uart:ash: ---> [FRAME type=ACK frmRx=2](ackRx=1)
debug 2024-09-08 21:39:21zh:ember:ezsp: <=== [CBFRAME: ID=69:"INCOMING_MESSAGE_HANDLER" Seq=80 Len=30]
debug 2024-09-08 21:39:21zh:ember:ezsp: ezspIncomingMessageHandler(): callback called with: [type=UNICAST], [apsFrame={"profileId":260,"clusterId":0,"sourceEndpoint":1,"destinationEndpoint":1,"options":320,"groupId":0,"sequence":162}], [packetInfo:{"senderShortId":15414,"senderLongId":"0xFFFFFFFFFFFFFFFF","bindingIndex":255,"addressIndex":255,"lastHopLqi":255,"lastHopRssi":-18,"lastHopTimestamp":0}], [messageContents=082b0b0081]
debug 2024-09-08 21:39:21zh:controller: Received payload: clusterID=0, address=15414, groupID=0, endpoint=1, destinationEndpoint=1, wasBroadcast=false, linkQuality=255, frame={"header":{"frameControl":{"frameType":0,"manufacturerSpecific":false,"direction":1,"disableDefaultResponse":false,"reservedBits":0},"transactionSequenceNumber":43,"commandIdentifier":11},"payload":{"cmdId":0,"statusCode":129},"command":{"ID":11,"name":"defaultRsp","parameters":[{"name":"cmdId","type":32},{"name":"statusCode","type":32}]}}
info 2024-09-08 21:39:21z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/0x3c6a2cfffed3a374', payload '{"current":5.138,"current_l1":1.709,"current_l2":1.712,"current_l3":1.717,"device":{"applicationVersion":0,"dateCode":"20230406","friendlyName":"0x3c6a2cfffed3a374","hardwareVersion":0,"ieeeAddr":"0x3c6a2cfffed3a374","manufacturerID":4412,"manufacturerName":"OWON Technology Inc.","model":"PC321","networkAddress":15414,"powerSource":"Unknown","stackVersion":0,"type":"Router","zclVersion":0},"energy":143.888,"energy_l1":47.994,"energy_l2":47.811,"energy_l3":48.083,"last_seen":"2024-09-08T19:39:02.101Z","linkquality":255,"power":458,"power_factor_l1":0.38,"power_factor_l2":0.38,"power_factor_l3":0.38,"power_l1":153,"power_l2":152,"power_l3":153,"power_reactive":1129,"reactive_energy":157.714,"reactive_energy_l1":52.312,"reactive_energy_l2":52.627,"reactive_energy_l3":52.775,"reactive_power_l1":375,"reactive_power_l2":377,"reactive_power_l3":377,"state":"ON","voltage_l1":237.1,"voltage_l2":237.2,"voltage_l3":237.1}'
`

Reply:
It still doesn't work? Can you try with `await endpoint.command(entity, command, payload, {disableDefaultResponse: true});`. We cannot use `const group = 0xFFE0;` as this would send the message (and thus resetting) all devices.

Reply:
No, it doesn't work.

What information can I use for 'entity' ?
>meta.logger.info(`Contenu de l'entity: ${JSON.stringify(entity)}`);

Result : 
`{"_events":{},"_eventsCount":0,"deviceID":13,"inputClusters":[0,3,1794],"outputClusters":[3],"profileID":260,"ID":1,"clusters":{"genBasic":{"attributes":{"zclVersion":0,"appVersion":0,"stackVersion":0,"hwVersion":0,"modelId":"PC321","manufacturerName":"OWON Technology Inc.","powerSource":0,"dateCode":"20230406"}},"seMetering":{"attributes":{"8451":850,"12547":4424,"16643":[0,9266],"24576":[0,0],"24577":[0,0],"24578":[0,0],"24579":[0,0],"multiplier":1,"divisor":1000,"currentSummDelivered":[0,10179],"instantaneousDemand":616,"status":0,"owonL1Energy":[0,3395],"owonL1PhasePower":206,"owonL1PhaseVoltage":2374,"owonL1PhaseCurrent":1473,"owonL1ReactiveEnergy":[0,3067],"owonL1PhaseReactivePower":283,"owonL1PowerFactor":59,"owonL2Energy":[0,3383],"owonL2PhasePower":205,"owonL2PhaseVoltage":2375,"owonL2PhaseCurrent":1472,"owonL2ReactiveEnergy":[0,3095],"owonL2PhaseReactivePower":283,"owonL2PowerFactor":59,"owonL3Energy":[0,3401],"owonL3PhasePower":206,"owonL3PhaseVoltage":2374,"owonL3PhaseCurrent":1479,"owonL3ReactiveEnergy":[0,3104],"owonL3PhaseReactivePower":284,"owonL3PowerFactor":59}},"genOnOff":{"attributes":{}}},"deviceIeeeAddress":"0x3c6a2cfffed3a374","deviceNetworkAddress":15414,"_binds":[{"cluster":1794,"type":"endpoint","deviceIeeeAddress":"0xe0798dfffeb8d7aa","endpointID":1},{"cluster":3,"type":"endpoint","deviceIeeeAddress":"0xe0798dfffeb8d7aa","endpointID":1},{"cluster":0,"type":"endpoint","deviceIeeeAddress":"0xe0798dfffeb8d7aa","endpointID":1},{"cluster":6,"type":"endpoint","deviceIeeeAddress":"0xe0798dfffeb8d7aa","endpointID":1}],"_configuredReportings":[],"meta":{},"pendingRequests":{"sendInProgress":false,"ID":1,"deviceIeeeAddress":"0x3c6a2cfffed3a374"}}`

Reply:
I see, thanks for checking!